### ISSUE TITLE
ChatHeads and Bubbles refuse to work

### ISSUE THREAD

**Issue by tamas646** [NeonOrbit/ChatHeadEnabler#16]
Today I noticed that the Messenger bubbles stopped working on my phone. It automatically closes and opens the Messenger app. I tried to switch to chatheads but it's behavior was the same:
**When I open chatheads/bubbles it immediately closes the chatheads/bubbles and opens the Messenger app.**

Clearing cache and storage did not solve the problem. Also disabling this module did not solve the problem.
The only way I could make chatheads working is to completely reinstall the app. But this way I lose the bubbles which I'd like to use.
If I change the chatheads to bubbles the problem comes up again.

There was no Android or Messenger app upgrade when the problem first occurred.

Any ideas what could cause this strange behavior?

Module version: 2.4.5
Messenger version: 446.0.0.44.109 (319209061)
Android version: 13 (LOS 20 2024-02-05)

**Comment by NeonOrbit** [comment]
> I tried to switch to chatheads but it's behavior was the same

Which behaviour was the same? You said this only happens when you try to open bubbles, not chathead.  
If the module can turn the chat head into a bubble option successfully, but your bubble isn't opening, then the bug is probably in LineageOS. If bubbles in other apps work fine, then it's likely an issue with Messenger.

**Comment by tamas646** [comment]
> Which behaviour was the same? You said this only happens when you try to open bubbles, not chathead.

Sorry for being unclear. Both bubbles and chathead refused to work until I reinstalled the app. But when I started using the module to switch between the two it started to crash again with both.

> If the module can turn the chat head into a bubble option successfully, but your bubble isn't opening, then the bug is probably in LineageOS. If bubbles in other apps work fine, then it's likely an issue with Messenger.

First I thought that the module causes these problems, this is why I wrote here.
But yesterday (with disabled module) Messenger automatically switched from chathead to bubbles and produced the same behavior as before (the bubbles closed and Messenger app opened).

So I can see now that it's indeed not a module bug.
Thank you for your help. I'm closing this issue...
